climada_petals/engine/supplychain.py

```diff
@@ -125,13 +127,21 @@ def read_wiod16(self, year=2014, range_rows=(5,2469),
 
         """
 
+        if not WIOD_DIRECTORY.exists():
+            WIOD_DIRECTORY.mkdir()
+
+            LOGGER.info('Downloading folder with WIOD tables')
+
+            downloaded_file_name = u_fh.download_file(WIOD_FILE_LINK,
+                                                      download_dir=WIOD_DIRECTORY)
+            downloaded_file_zip_path = Path(downloaded_file_name + '.zip')
+            Path(downloaded_file_name).rename(downloaded_file_zip_path)
+
+            with zipfile.ZipFile(downloaded_file_zip_path, 'r') as zip_ref:
+                zip_ref.extractall(WIOD_DIRECTORY)
+
```

Comment:
it seems like the WIOD_DIRECTORY is created when I import the supply chain module. So the check for the directory passes and it is not downloading the file. And further down it fails. So I think that you should rather test if the file exists rather than the directory.